agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -392,7 +392,7 @@ private static Set<Feature> initStatsbeatFeatureSet(Configuration config) {
     if (!config.preview.instrumentation.jaxrsAnnotations.enabled) {
       featureList.add(Feature.JAXRS_ANNOTATIONS_DISABLED);
     }
-    if (config.preview.javaScriptSnippet.enabled) {
+    if (config.preview.browserSdkLoader.enabled) {
       featureList.add(Feature.JAVASCRIPT_SNIPPET);
```

Comment:
let's rename the "feature" also
```suggestion
      featureList.add(Feature.BROWSER_SDK_LOADER);
```